woocommerce/woocommerce-ios
# Products List: Replace the word "variant" with "variation"

Closes: #5644

### Description
This PR replaces the usage of the word "variant" with "variation" by making the following changes.

- Replace word "variant" with "variation" in view model.
- Move these `NSLocalizedString`s to private enum.
- Update tests to expect the word "variation".

### Testing instructions
1. Ensure that you have two products X and Y. 
1. Product X should have exactly 1 variation. This helps us to test the singular version of the word "variant".
1. Product Y should have more than 1 variation. This helps us to test the plural version of the word "variant" which is "variations".
1. Navigate to the Products tab to view the list of products.
1. Observe that the label in product X reads as "variation"
1. Observe that the label in product Y reads as "variations"


### Screenshots

| Before | After |
| --- | --- |
| <img src="https://user-images.githubusercontent.com/524475/148504707-caa27d83-c18d-4c6e-bdf1-1c876e09d95d.png" width="350"> | <img src="https://user-images.githubusercontent.com/524475/148504836-dd84659d-54f2-4c41-a947-3761bc1464e3.png"  width="350"> |


---
